### DISCUSSION TITLE
Filterfunktion im Widget

### DISCUSSION THREAD

**Discussion by tschortschde** [Zefau/ioBroker.jarvis#2426] [Q&A]
Hallo,
leider komme ich mit der Anleitung zu der Filterfunktion nicht klar ;-(
Wie kann ich die Werte (Helligkeit, Windgeschwindigkeit und  Temperatur ) immer angezeigt bekommen und die Werte Windwarnung und Regen nur wenn der State im iob auf "true" steht?
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/c8e38b2a-df02-4968-8460-afa7d8112154)
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/1be21e40-59c4-49a3-9b75-b555964d6574)

**Comment by mcuiobroker** [comment]
Würde momentan mit v3.1.9-beta.2 nicht funktionieren, da wenn man den Eintrag nicht mehr sehen will ( = false) dieser nicht ausgeblendet wird. Man müsste dann F5 drücken, dann würde es korrekt angezeigt werden, ist aber nicht sinnvoll.

Hab es mal hier erklärt.
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/module/statelist/eintraege-filtern#beispiel-fuer-states-wo-nur-bestimmte-dauerhaft-angezeigt-werden-sollen

siehe auch https://github.com/Zefau/ioBroker.jarvis/issues/2430

**Reply by tschortschde** [reply]
Danke,  freue mich darauf sobald es wieder funktioniert😉
